lib/MusicBrainz/Server/Controller/Release.pm

```diff
@@ -109,6 +112,8 @@ after 'load' => sub {
         $c->model('CritiqueBrainz')->load_display_reviews($release->release_group)
             unless $returning_jsonld;
     }
+
+    $c->stash->{may_have_discids} = $release->may_have_discids;
```

Comment:
It might be a bit cleaner to just use `release.may_have_discids` in the templates (which should work without any changes in TT, but for the JS we'd have to serialize it in `TO_JSON`). Though you may have some other reason for adding it to the stash that I've missed.

Reply:
This is much better in other ways, but it breaks Flow, since in EntityTabs all we know is "CoreEntityT" (that has discids = true, but that doesn't help Flow). Suggestions?